ckanext/switzerland/i18n/de/LC_MESSAGES/ckanext-switzerland.po

```diff
@@ -717,43 +717,90 @@ msgstr "Es gibt Neuigkeiten!"
+msgid "Homepage"
 msgstr "Startseite"
 
-#: ckanext/switzerland/templates/admin/config.html:18
+#: ckanext/switzerland/templates/admin/config.html:21
+msgid "Instruction how to set the text"
+msgstr "Anleitung für das Setzen des Textes"
+
+#: ckanext/switzerland/templates/admin/config.html:30
+msgid "Important"
+msgstr "Wichtig"
+
+#: ckanext/switzerland/templates/admin/config.html:31
 msgid ""
-"When it is set a link will be generated to point to it on each page other "
-"than the homepage."
-msgstr "Wenn gesetzt wird auf jeder Seite ausser der Startseite ein Link zu "
-"den Neuigkeiten eingefügt."
+"Currently all 4 languages are need to be set. The text for each language will"
+" appear on the locale of the homepage for that language."
+msgstr "Aktuell müssen alle 4 Sprachen gesetzt werden. Der Text jeder Sprache "
+"wird dann in der jeweiligen Lokale der Startseite angezeigt."
+
+#: ckanext/switzerland/templates/admin/config.html:34
+msgid "Standard CKAN settings"
+msgstr "CKAN Standard Einstellungen"
+
+#: ckanext/switzerland/templates/admin/config.html:36
+msgid ""
+"This is the title of this CKAN instance It appears in various places "
+"throughout CKAN."
+msgstr "Der Titel der CKAN Instanz wird an verschiedenen Stellen auf "
+"der CKAN Instanz angezeigt."
+
+#: ckanext/switzerland/templates/admin/config.html:39
+msgid ""
+"Choose from a list of simple variations of the main colour scheme to get a "
+"very quick custom theme working."
+msgstr "Hier kann ein simples Farbschema für die CKAN Instanz "
+"gewählt werden."
+
+#: ckanext/switzerland/templates/admin/config.html:43
+msgid ""
+"This is the logo that appears in the header of all the CKAN instance "
+"templates."
+msgstr "Hier kann der Seitenslogan eingegeben werden, der dann auf jeder "
+"Seite erscheint."
+
+#: ckanext/switzerland/templates/admin/config.html:46
+msgid "This text will appear on this CKAN instances, see "
+msgstr "Dieser Text erscheint auf dieser CKAN Instanz, siehe "
+
+#: ckanext/switzerland/templates/admin/config.html:47
+msgid "About page"
+msgstr "Über uns Seite"
+
+#: ckanext/switzerland/templates/admin/config.html:48
+msgid "Custom CSS"
+msgstr "Custom CSS"
+
+#: ckanext/switzerland/templates/admin/config.html:49
+msgid "This can be used for custom CSS on a CKAN site."
+msgstr "Diese Einstellung kann für CSS Definitionen benutzt werden."
 
-#: ckanext/switzerland/templates/admin/config.html:20
+#: ckanext/switzerland/templates/admin/config.html:52
 msgid ""
-"Use Markdown to write it: Use subheaders #de, #fr, #en, #it to add different "
-"texts for each language."
-msgstr "Benutze Markdown um den Text zu schreiben: über die Unterüberschriften "
-"#de, #fr, #en, #it lässt sich steuern, welcher Text in jeder Spachversion "
-"erscheinen soll."
\ No newline at end of file
+"This is for choosing a predefined layout for the modules that appear on your "
+"homepage."
+msgstr "Mit dieser Einstellung kann ein vordefiniertes Layout für die "
+"Starseite gewähtl werden."
```

Comment:
For the text that was already translated in CKAN, can we use the existing translation instead of creating our own? (I know I corrected a missing `.` above that was also wrong in the existing CKAN translation, but I think that's a reasonable change. :slightly_smiling_face: )